targets/targets.json

```
@@ -234,7 +243,15 @@
             "SPISLAVE",
             "STDIO_MESSAGES"
         ],
-        "default_lib": "small",
+        "selected_c_libs": {
+            "arm": "small",
+            "gcc_arm": "small"
+        },
+        "supported_c_libs": {
+            "arm": ["std", "small"],
+            "gcc_arm": ["std", "small"],
```

```suggestion
            "gcc_arm": ["small"],
```
Did you check that it can be built and ran fine with GCC's newlib? If it does not work then it should be what is suggested above. This comment is to be also considered for the changes in the other targets where `"default_lib": "small"` was replaced by `selected_c_libs` and `"supported_c_libs": {"gcc_arm": ["std", "small"], ...}`